index.js

```diff
@@ -81,7 +92,7 @@ async function newReconciliation(firmId, handle) {
       return;
     }
     const template = await ReconciliationText.read(handle);
-    template.version_comment = "Updated through the CLI";
+    template.version_comment = message;
```

Comment:
[TO CONSIDER] Just to confirm, but would this message be visible anywhere as I checked the changelog and it seems only the update messages are visible? (However wouldn't do any harm to retain this code just in case)
![6A81BA1E-650A-489A-B84A-2FE00AD7E29F](https://github.com/silverfin/silverfin-cli/assets/128508661/bfd9e8fe-376c-4b43-85bc-2f72b853b3d1)

Reply:
Good point, we don't need the message while creating :slightly_smiling_face: